recipes-tag/librdkafka/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "librdkafka" %}
+{% set version = "0.11.6" %}
```

I think we can use v1.0.1 here. The corresponding conda-forge feedstock PR is passing: https://github.com/conda-forge/librdkafka-feedstock/pull/33.
```suggestion
{% set version = "1.0.1" %}
```